canvas_account_admin_tools/settings/base.py

```diff
@@ -263,9 +263,17 @@
 ICOMMONS_REST_API_TOKEN = SECURE_SETTINGS.get('icommons_rest_api_token')
 ICOMMONS_REST_API_SKIP_CERT_VERIFICATION = False
 
-PERMISSION_ACCOUNT_ADMIN_TOOLS = 'account_admin_tools'
-PERMISSION_PEOPLE_TOOL = 'people_tool'
-PERMISSION_XLIST_TOOL = 'cross_listing'
+# TOOL_PERMISSIONS is used to map app names to SchoolPermission.permission names
+# to determine visibility of tools on dashboard and access to tool views/pages
+TOOL_PERMISSIONS = {
+    # NOTE: canvas_account_admin_tools permission name is shortened to
+    # account_admin_tools due to SchoolPermission.permission field max length
+    'canvas_account_admin_tools': 'account_admin_tools',  # dashboard
+    'conclude_courses': 'conclude_courses',  # external tool
+    'course_info': 'course_info',
+    'cross_list_courses': 'cross_list_courses',
```

Comment:
Yeah, if we're going to have to be backwards-compatible, we will need to use the existing LtiPermission names, or at least map them somehow. This code was written assuming we could use new names, before our discussion about automatically mapping them. Good catch.